### PR TITLE
Fix conflict for stage testing and live content

### PR DESCRIPTION
**Pull request by petr-balogh**
Fixes: #1335
Signed-off-by: Petr Balogh <pbalogh@redhat.com>

### CODE REVIEW

**Comment by petr-balogh** [comment]
Verification job running here:  https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/3842/console

**Comment by petr-balogh** [comment]
Had still few issues. Fixed them and running other verification just OCS deployment:
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/3855/console

**Comment by petr-balogh** [comment]
There is still some issue and from some reason catalog source is in connecting state.
When I tried locally delete operatorsource and added again it sometime works and sometime not. Will probably have to raise some OCP BZ for that but last time it seems to connected but found another issue:

```
oc get packagemanifest  -n openshift-marketplace --selector=ocs-operator-internal=true
NAME                             CATALOG                       AGE
3scale-operator                  Openshift Container Storage   14m
amq-broker                       Openshift Container Storage   14m
amq-broker-rhel8                 Openshift Container Storage   14m
amq-online                       Openshift Container Storage   14m
amq-streams                      Openshift Container Storage   14m
amq7-cert-manager                Openshift Container Storage   14m
amq7-interconnect-operator       Openshift Container Storage   14m
businessautomation-operator      Openshift Container Storage   14m
cam-operator                     Openshift Container Storage   14m
cluster-logging                  Openshift Container Storage   14m
codeready-workspaces             Openshift Container Storage   14m
datagrid                         Openshift Container Storage   14m
dv-operator                      Openshift Container Storage   14m
eap                              Openshift Container Storage   14m
elasticsearch-operator           Openshift Container Storage   14m
fuse-apicurito                   Openshift Container Storage   14m
fuse-camel-k                     Openshift Container Storage   14m
fuse-online                      Openshift Container Storage   14m
jaeger-product                   Openshift Container Storage   14m
kiali-ossm                       Openshift Container Storage   14m
kubevirt-hyperconverged          Openshift Container Storage   14m
local-storage-operator           Openshift Container Storage   14m
metering-ocp                     Openshift Container Storage   14m
nfd                              Openshift Container Storage   14m
ocs-operator                     Openshift Container Storage   14m
openshiftansibleservicebroker    Openshift Container Storage   14m
openshifttemplateservicebroker   Openshift Container Storage   14m
serverless-operator              Openshift Container Storage   14m
servicemeshoperator              Openshift Container Storage   14m
sriov-network-operator           Openshift Container Storage   14m
```

There are more operators available from stage registry and make troubles for selector.

**Comment by vasukulkarni** [comment]
> There is still some issue and from some reason catalog source is in connecting state.
> When I tried locally delete operatorsource and added again it sometime works and sometime not. Will probably have to raise some OCP BZ for that but last time it seems to connected but found another issue:
> 
> ```
> oc get packagemanifest  -n openshift-marketplace --selector=ocs-operator-internal=true
> NAME                             CATALOG                       AGE
> 3scale-operator                  Openshift Container Storage   14m
> amq-broker                       Openshift Container Storage   14m
> amq-broker-rhel8                 Openshift Container Storage   14m
> amq-online                       Openshift Container Storage   14m
> amq-streams                      Openshift Container Storage   14m
> amq7-cert-manager                Openshift Container Storage   14m
> amq7-interconnect-operator       Openshift Container Storage   14m
> businessautomation-operator      Openshift Container Storage   14m
> cam-operator                     Openshift Container Storage   14m
> cluster-logging                  Openshift Container Storage   14m
> codeready-workspaces             Openshift Container Storage   14m
> datagrid                         Openshift Container Storage   14m
> dv-operator                      Openshift Container Storage   14m
> eap                              Openshift Container Storage   14m
> elasticsearch-operator           Openshift Container Storage   14m
> fuse-apicurito                   Openshift Container Storage   14m
> fuse-camel-k                     Openshift Container Storage   14m
> fuse-online                      Openshift Container Storage   14m
> jaeger-product                   Openshift Container Storage   14m
> kiali-ossm                       Openshift Container Storage   14m
> kubevirt-hyperconverged          Openshift Container Storage   14m
> local-storage-operator           Openshift Container Storage   14m
> metering-ocp                     Openshift Container Storage   14m
> nfd                              Openshift Container Storage   14m
> ocs-operator                     Openshift Container Storage   14m
> openshiftansibleservicebroker    Openshift Container Storage   14m
> openshifttemplateservicebroker   Openshift Container Storage   14m
> serverless-operator              Openshift Container Storage   14m
> servicemeshoperator              Openshift Container Storage   14m
> sriov-network-operator           Openshift Container Storage   14m
> ```
> 
> There are more operators available from stage registry and make troubles for selector.

I seriously think we should raise an bz issue with OCP here, Since you are seeing it now I recommend you raise it.

**Comment by petr-balogh** [comment]
Another verification job running here:
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/3921/console

**Comment by petr-balogh** [comment]
Verification job from scratch running here: https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/3923/console

Verified just OCS deployment here:
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/3921/

Regular deployment testing will run with regular PR verification job.

**Comment by petr-balogh** [comment]
I see that deployment started successfully and performing well:  https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/3931/console marking as verified as it will finish deployment soon as it's already deal with monitoring stuff.

File: ocs_ci/ocs/resources/packagemanifest.py [hunk 1]
```diff
@@ -37,11 +37,23 @@ def __init__(
         )
 
     def get(self, **kwargs):
+        resource_name = kwargs.get("resource_name", "")
```
Comment (clacroix12) [inline ocs_ci/ocs/resources/packagemanifest.py]: Why not just use `self.resource_name` as the default value here and remove line 41?

```python
resource_name = kwargs.get("resource_name", self.resource_name)
```

Comment (petr-balogh) [reply ocs_ci/ocs/resources/packagemanifest.py]: yep we could :) if I will touch this file I can update this logic here

**Pull request merged**